### ISSUE TITLE
Typescript is incorrectly specified as devDependency in package.json

### ISSUE THREAD

**Issue by LeonardDrs** [PSeitz/ts-dependency-graph#1]
Hello, 

Tried to use this module with `npx` but it fails early with 
```
$ npx ts_dependency_graph --start src/index.ts
npx: installed 38 in 2.519s
Cannot find module 'typescript'
Require stack:
- /home/ldrouillas/.npm/_npx/1223903/lib/node_modules/ts_dependency_graph/dist/scan/imports.js
- /home/ldrouillas/.npm/_npx/1223903/lib/node_modules/ts_dependency_graph/dist/index.js

```
And when we look at https://github.com/PSeitz/ts-dependency-graph/blob/master/src/scan/imports.ts#L3, we indeed import typescript in the code, but we specified `typescript` as ` devDependency`: https://github.com/PSeitz/ts-dependency-graph/blob/master/package.json#L35, which explains the issue with `npx`.

**Comment by PSeitz** [comment]
Thanks for the bug report, I will provide a fix for this shortly.
In the meantime you can use it when installing typescript globally

`npm install -g typescript`

**Comment by PSeitz** [comment]
okay should be fixed with version 1.1.3
